analysis.json

```diff
@@ -21,6 +21,40 @@
           }
         }
       }
+    },
+    "CVE-2022-1471": {
+      "dependency": {
+        "name": "org.yaml:snakeyaml",
+        "affectedVersions": [ "[,2.0)" ]
+      },
+      "assessments": {
+        "com.vaadin:vaadin-spring-boot-starter": {
+          "affectedVersions": {
+            "[14.0.0,23.3.5)": {
+              "status": "TRUE_POSITIVE",
+              "comment": "The snakeyaml dependency is a transitive dependency of the spring-boot-dependencies 2.x-3.1 dependency. Vaadin project is not depending on the vulnerable dependency (org.yaml:snakeyaml) directly, users can add the dependency if needed.",
+              "fixedIn": "23.3.5"
+            }
+          }
+        }
+      }
+    },
+    "CVE-2023-4586": {
+      "dependency": {
+        "name": "io.netty/netty-handler",
```

Comment:
Lets use the same `group:name` pattern here too.

Reply:
done